### DISCUSSION TITLE
Having some trouble importing

### DISCUSSION THREAD

**Discussion by namuuu** [pdfme/pdfme#164] [Q&A]
Hello, I just started trying to use pdfme for a project, and I have some trouble using tsc to compile it.

As of right now, I'm using `import { Template, generate } from "@pdfme/generator"` to import everything

But I'm getting an error `Namespace "path" has no exported member 'extendShape'` 57 times while compiling it. As I'm new to TypeScript, this could be a fairly simple problem, but I can't seem to find an answer.

Thanks for the help, and I can provide more information if needed.

package.json:
`{
  "dependencies": {
    "@pdfme/generator": "1.1.10"
  },
  "devDependencies": {
    "@types/node": "^18.16.1",
    "typescript": "^5.0.4"
  }
}`

I almost have nothing in my ts file.

**Comment by hand-dot** [comment]
Hi @namuuu 
Recently I publish 1.1.10, maybe this version makes the same error.

Pls use 1.1.9

Thanks
